vortex-api/src/main/java/mil/army/usace/hec/vortex/VortexGrid.java

```diff
@@ -77,6 +77,32 @@ public static class VortexGridBuilder {
         private ZonedDateTime endTime;
         private Duration interval;
         private VortexDataType dataType;
+        private VortexVariable variable;
```

Comment:
Why did we add `VortexVariable` as a field? It seems like there is opportunity for conflict with other state variables like `short_name` which is essentially a string representation of a variable.